### ISSUE TITLE
File watching on mounted volumes doesn't work

### ISSUE THREAD

**Issue by whitecolor** [docker/for-win#362]
### Expected behavior
For example these popular packages: 
`tsc --watch` (typescript compiler)
`nodemon`
`node-dev`
`webpack`

should be able to watch file changes on mounted volumes.

### Actual behavior
tsc --watch (doesn't work on mounted volumes)
nodemon (works too only using legacy flag)
[node-dev](https://github.com/fgnass/node-dev/issues/190) (works only with legacy `--poll` option)

Will it be possible to fix somehow or have to find other solutions (for example for `typescript`)?

**Comment by friism** [comment]
Thanks for reporting, this is a known limitation: https://docs.docker.com/docker-for-windows/troubleshoot/#inotify-on-shared-drives-does-not-work

**Comment by whitecolor** [comment]
Ok, it seems that `tsc` works, not sure why it seem to me that it doesn't.

**Comment by nhooey** [comment]
It looks like [inotify-proxy](https://github.com/sillypog/inotify-proxy) is a workaround to this problem.

**Comment by elonderin** [comment]
## update: 

### TLDR
- still not working as of 2.2.0.3
- workaround is: use install docker-windows-volume-watcher

### long story

- i found this thread: https://forums.docker.com/t/file-system-watch-does-not-work-with-mounted-volumes/12038
- and installed it: `pip install docker-windows-volume-watcher`
- seems to work (even on my corporate Microsoft Windows [Version 10.0.17134.1304] )
- has a bit of a delay 

### the link to the docker docs is dead
> 
> Thanks for reporting, this is a known limitation: https://docs.docker.com/docker-for-windows/troubleshoot/#inotify-on-shared-drives-does-not-work

i'd wish, i had seen that limitation beforehand :sigh:

**Comment by elonderin** [comment]
# update
seems to be fixed in 2.2.0.4

**Comment by trajano** [comment]
Seems to be broken in 2.2.0.5

I tied 
```yaml
  vue:
    image: node:lts
    networks:
      - intranet
    volumes:
      - /d/p/spring-cloud-demo/vue-app:/work
    command: >
      bash -c "cd /work
      && npm run serve"
```

Then tried changing a file and does not work.
